src/tx/validator.ts

```diff
@@ -33,27 +65,41 @@ const validators = [
       checkedKeys: ['encodedTx', 'signatures']
     }]
   },
-  ({ encodedTx, tx }, { node, parentTxTypes, txType }) => {
+  ({
+    encodedTx,
+    tx
+  }: {
+    encodedTx: EncodedTx<TxParams>
+    tx: TxSigned
+  }, {
```

Comment:
Here should be a common type to extract, all validators called with the same arguments